samples/zigbee/light_switch/README.rst

```diff
@@ -105,18 +105,29 @@ You can find the configuration files in the :file:`samples/zigbee/light_switch`
 Activating optional extensions
 ------------------------------
 
-To activate the optional extensions supported by this sample, modify :makevar:`OVERLAY_CONFIG` as follows:
+To activate the :ref:`lib_zigbee_fota`, use the :file:`prj_fota.conf` configuration file and add the ``-DCONF_FILE='prj_fota.conf'`` flag to the build command.
```

Comment:
```suggestion
To activate the :ref:`lib_zigbee_fota`, use the :file:`prj_fota.conf` configuration file.
```
Alternatively:
```suggestion
To activate the :ref:`lib_zigbee_fota`, add the ``-DCONF_FILE='prj_fota.conf'`` flag to the build command to use the :file:`prj_fota.conf` configuration file.
```
Mentioning the flag isn't necessary since it comes up in the example command below.

Reply:
I prefer the first, simplified version.